### DISCUSSION TITLE
i have problem with my checkout page

### DISCUSSION THREAD

**Discussion by niagaku909** [woocommerce/woocommerce#52251] [Checkout Flow]
![Gambar WhatsApp 2024-10-23 pukul 09 31 36_9079f037](https://github.com/user-attachments/assets/bd7a2c8f-5768-4dac-a2a3-b97fabe75c3e)

like that,,how i must to fix it

**Comment by juliaamosova** [comment]
Closing as duplicate of https://github.com/woocommerce/woocommerce/discussions/52249.
